test/files/jvm/future-spec/TryTests.scala

```diff
@@ -126,5 +126,27 @@ class TryTests extends MinimalScalaTest {
         }
       }
     }
+
+    "contains" in {
+      Success(1) contains 1 mustEqual true
+      Success(2) contains 1 mustEqual false
+      Failure(new Exception) contains 1 mustEqual false
+    }
+
+    "exists" in {
+      Success(1).exists(_ > 0) mustEqual true
+      Success(2).exists(_ < 1) mustEqual false
+
+      Failure(new Exception).exists(_ => true) mustEqual false
+      Failure(new Exception).exists(_ => false) mustEqual false
+    }
+
+    "forall" in {
+      Success(1).forall(_ > 0) mustEqual true
+      Success(2).forall(_ < 1) mustEqual false
+
+      Failure(new Exception).exists(_ => true) mustEqual true
+      Failure(new Exception).exists(_ => false) mustEqual true
+    }
```

Comment:
There is a unit test under `test/junit/scala/util/TryTest.scala`. Note that junit `assertEquals(expected, actual)`, don't copy the style there. (If you care.) Moving this test to that location would be Eagle Scout level Scouting. (Note that I refrain from calling it Boy Scouting in this day and age.)

Reply:
refactor those tests https://github.com/scala/scala/pull/8455